documentation/Migration/Overview.md

```diff
@@ -3,5 +3,4 @@
 These documents outline how to update your code due to breaking changes between major versions.
 
 - [Updating to 0.109.*](./Update-0.109.md)
-- [Updating to 1.0.0-rc](./Update-1.0.0-rc.md)
```

Comment:
There is already documentation in here from a previous check-in. That will need to be moved to the new rc1.md file.

Reply:
Good catch. Merged the files